### ISSUE TITLE
Risk check failed with error 16 , followed by error 13 

### ISSUE THREAD

**Issue by ndegendogo** [corona-warn-app/cwa-app-ios#3970]
## Avoid duplicates
* [ ] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [ ] Bug is not already reported in another issue

## Technical details

- Device name: iPhone 6
- iOS version: iOS 12.5.5
- App version: cwa 2.14.1

## Describe the bug

I just opened the cwa to check my risk status, but instead saw the error message (Fehler 13). See screenshot.
The timestamp (23:43 UTC) shows that the ~risk check~ error message was triggered at the moment when I opened the app (00:43 local time).
The error message below shows that the last successful check was more than 24 hours ago (vorgestern 05:42).
This matches with the ENF log.
Note: The device has no SIM card. Today it had WLAN coverage the whole day, so I would have expected risk checks.
Note: I can provide my error log, in case you are interested.

## Steps to reproduce the issue
- open the app
- see error message for failed risk check

## Additional context

![20211212_error13](https://user-images.githubusercontent.com/6118267/145695689-774030b9-3623-4721-98ef-bbf61c3a7827.jpeg)

---
Internal Tracking ID: [EXPOSUREAPP-11005](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11005)

**Comment by ndegendogo** [comment]
My ENF log as screenshot ...

![20211212-ENF](https://user-images.githubusercontent.com/6118267/145695921-bb79e644-c561-4599-81dc-c5ec630b1f03.png)

**Comment by ndegendogo** [comment]
... and ENF log as JSON file
[ExposureChecks-2021-12-12.json.txt](https://github.com/corona-warn-app/cwa-app-ios/files/7697938/ExposureChecks-2021-12-12.json.txt)

**Comment by ndegendogo** [comment]
The error log file shows ENError 16 (Database inaccessible) at 2021-12-11T01:23:22Z. 
After this cwa retries several times at 04:43:54 / 04:43:58 / 08:40:50 / 12:38:36 / 16:41:39, all with error 16.
Finally it retries at 20:39:19 and from now on it gets error 13 (Max per-day API limit reached: 6).

So, the root cause for the failing risk check seems to be error 16, but this is not shown at the user interface, only in the error log.

**Comment by ndegendogo** [comment]
[Apple documentation for ENError 16](https://developer.apple.com/documentation/exposurenotification/enerror/code/datainaccessible) tells us:
"Declaration
case dataInaccessible = 16
ENError.Code.dataInaccessible
The user must unlock the device before it can access data."

***What does this mean?***

**Comment by dsarkar** [comment]
Good morning @ndegendogo Thanks: Internal Tracking ID: [EXPOSUREAPP-11005](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11005)


---
Corona-Warn-App Open Source Team

**Comment by thomasaugsten** [comment]
Officially this means to access the secure enclave after a reboot you have to enter your passcode, to access ENF information.
We saw this error also on unlocked device,  the recommendation was to not show this error to the user because it will recover automatically. Be aware on iOS 12.5 only once a day is a check by iOS guaranteed.

**Comment by ndegendogo** [comment]
Thanks @thomasaugsten.
And, yes, you are right. It has indeed recovered, and I am not aware of any special actions on my side except that I waited patiently.
😁😅

Still one question remains:
In contrary to our understanding, this error seems to have counted against the rate limit. See also [this discussion](https://github.com/corona-warn-app/cwa-app-ios/pull/3284#issuecomment-890104094) "The feedback from Apple at the beginning of the project was that errors are not count into the quota."

*So, maybe that information is meanwhile outdated? Would it be a candidate for clarification with Apple?*

**Comment by Ein-Tim** [comment]
Is it still necessary to have this issue open?

**Comment by ndegendogo** [comment]
@Ein-Tim thanks for grooming through all the old tickets!
The new solution to handle (or rather suppress) error 13 fixes this issue well from a user's perspective.

Closing as resolved. 🎉🎉

**Comment by dsarkar** [comment]
@ndegendogo Thanks for reporting and feedback. @Ein-Tim Thanks again for "house keeping"! Best wishes, DS
